jupyterhub/templates/proxy/deployment.yaml

```diff
@@ -53,11 +53,11 @@ spec:
           image: {{ .Values.proxy.chp.image.name }}:{{ .Values.proxy.chp.image.tag }}
           command:
             - configurable-http-proxy
-            - --ip=0.0.0.0
-            - --api-ip=0.0.0.0
+            - "--ip=::"
+            - "--api-ip=::"
```

Comment:
How will this behave on an ipv4-only cluster?

Reply:
It has no effects for IPv4-only clusters.

Binding to `::` actually causes to socket to listen for IPv4 and IPv6 unless the `IPV6_V6ONLY` socket option is added to the socket which is normally not the case by default.

See also: http://man7.org/linux/man-pages/man7/ipv6.7.html